network/a10/a10_service_group.py

```diff
@@ -60,6 +83,22 @@
         specify the C(status:). See the examples below for details.
     required: false
     default: null
+  write_config:
```

Comment:
Just checking you saw that `write_config` and `validate_certs` docs need removing as well